### PR TITLE
UI: custom onboard styles

### PR DESCRIPTION
**Pull request by kangarang**
this pr fixes mobile-responsiveness (TokenWrapper) and adds custom styles to the onboard.js modal

### CODE REVIEW

**Comment by miguelmota** [comment]
This looks perfect

2 minor things:

The text down here is very small and unreadable. Maybe we can remove it or enlarge it

DeepinScreenshot_select-area_20220103175839

All 3 options can be shown by default to remove the show more button (something lito mentioned)

DeepinScreenshot_select-area_20220103175910

File: packages/frontend/src/components/header/Header.tsx [hunk 1]
```diff
@@ -191,3 +208,6 @@ const Header: FC = () => {
 }
 
 export default Header
+
+// <div className={styles.balancePill} title={connectedNetwork?.name}>
```
Comment (miguelmota) [inline packages/frontend/src/components/header/Header.tsx:212]: commented out code can be removed

**Comment by kangarang** [comment]
appreciate the review @miguelmota! will address your comments

**Comment by kangarang** [comment]
@miguelmota, strange, i do not see the tiny "powered by blocknative" text in my local env.

Screen Shot 2022-01-03 at 11 09 15 PM

could you inspect that element and send a screeshot pls? i'll see if i can rm that element

**Comment by miguelmota** [comment]
@kangarang the `.bn-branding` div

DeepinScreenshot_select-area_20220103202034

**Comment by kangarang** [comment]
@miguelmota since i could not reproduce, i tried 2 different approaches. lemme know if it still shows up!

**Comment by miguelmota** [comment]
looks fixed!

DeepinScreenshot_select-area_20220103204429

**Pull request merged**